### DISCUSSION TITLE
Estimated running time

### DISCUSSION THREAD

**Discussion by snayfach** [ablab/spades#1158] [Q&A]
Does anyone know of any published running time statistics for metaspades? I'd like to estimate the average running time (per million reads) across different metagenomes.

**Comment by asl** [comment]
@snayfach The variance would be too big for any reasonable estimates. The running time (and memory consumption) depends on many factors including the genome size, repeat content, error rate, strain heterogeneity, etc. So, it's not all about the # reads... After all, the same million of reads might contain 2 species, and might – 200.

**Comment by snayfach** [comment]
That's fair. I'll mark this as resolved, but have you guys ever tried to estimate the running time based on the k-mer distribution from a random sample of reads?

**Reply by asl** [reply]
We do not. However I heard that there were some approaches here with quite mixed results.
